utils/EventRepository.js

```diff
@@ -226,14 +69,16 @@ class EventRepository {
     }
 
     // modify the url for event specific endpoints
-    const url = this.url.slice(-1) == "/" ? this.url.slice(0, -1) : this.url;
-    // add items to the queue
-    this.payloadQueue.addItem({
-      url: `${url}/v1/${type}`,
-      headers,
-      message,
-    });
+    const url = this.url.slice(-1) === "/" ? this.url.slice(0, -1) : this.url;
```

Comment:
Why does this have to be done for each queue item?

Reply:
done